cves/CVE-2011-1772.yml

```diff
@@ -119,20 +119,20 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
+  answer: This bug was discovered by Dr. Marian Ventuneac at Genworth Financial, likely through either vulnerability testing or a previous attack. After looking at the Jira entry and Apache releases, it is not clear exactly how this vulnerability was discovered.
```

Comment:
You can add multiline comments as follows:
answer: |
      My multiline
      comment here

Reply:
I wasn't aware of this, thanks! It definitely makes this a lot more readable.